src/projects.h

```diff
@@ -685,7 +688,7 @@ int pj_angular_units_set(paralist *, PJ *);
 paralist *pj_clone_paralist( const paralist* );
 paralist *pj_search_initcache( const char *filekey );
 void      pj_insert_initcache( const char *filekey, const paralist *list);
-paralist *pj_get_init(projCtx ctx, paralist **start, paralist *next, char *name, int *found_def);
+paralist *pj_get_init(projCtx ctx, paralist *start, char *key);
```

Comment:
This breaks public "API" :(

It's an undocumented part of the `projects.h` "API", so at least we need to discuss if this should be allowed to change or not. Since `projects.h` is a somewhat loose definitions of an API I think changes to functions that are mostly there for internal use should be allowed.

Reply:
At least, I believe that anyone outside the PROJ developer group, who would be able to make use of this in an application program, really should step forward and join the fun as PROJ developer :-)

As mentioned in #678, the `pj_`-namespace is heavily overloaded, and deciding what part can be considered API, and what cannot, is probably a matter of balanced judgment. In this case, I believe the risk of stepping on someones code is minimal - and at least, I believe that anyone making sense of `pj_get_init` in an application context, would probably have no problems migrating.

Reply:
For extra safety, and because it better reflects the actual effect of the function, I have changed the name to `pj_expand_init` and added an extensive comment describing the action of the function.

This will make it *fail early and noisily* for anyone who may have used pj_get_init, and provide the necessary details for migrating.